### ISSUE TITLE
aenum/__init__.py:220: DeprecationWarning: invalid escape sequence \w

### ISSUE THREAD

**Issue by arielf** [ethanfurman/aenum#2]
#### In python3.8, under pytest, I started seeing the following warning:

```
... /lib/python3.8/site-packages/aenum/__init__.py:220
... /lib/python3.8/site-packages/aenum/__init__.py:220: DeprecationWarning: invalid escape sequence \w
```

#### Trivial fix:

Change line 220 in `aenum/__init__.py`  to use a raw string (`r` prefix):
```
pattern = r'^_%s__\w+[^_]_?$' % (cls_name, )
```

Thank you.

**Comment by ethanfurman** [comment]
Thanks.  I need to change all my `re` strings to `raw`.

**Comment by ethanfurman** [comment]
It will be fixed in 3.0, which should be out a couple days.

**Comment by ethanfurman** [comment]
Released.
